auction-app-backend/src/main/java/com/example/auctionapp/service/implementation/AuthServiceImpl.java

```diff
@@ -70,6 +71,10 @@ public JwtResponse signIn(final LoginRequest loginRequest) {
         final UserEntity user = userRepository.findUserEntityByEmail(loginRequest.getEmail())
                 .orElseThrow(() -> new ResourceNotFoundException("User not found"));
 
+        if(!user.getActive()) {
+            throw new InvalidAccount("This account has been deactivated");
```

Comment:
Better name for this exception would be DeactivatedAccount or something like that, InvalidAccount is generic